src/goSuggest.ts

```diff
@@ -44,6 +44,14 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 		return this.provideCompletionItemsInternal(document, position, token, vscode.workspace.getConfiguration('go'));
 	}
 
+	private isInBlockComment(document: vscode.TextDocument, position: vscode.Position): boolean {
+		let text = document.getText();
+		let currentOffset = document.offsetAt(position);
+		let lastBlockCommentStartIndex = text.lastIndexOf('/*', currentOffset);
+		let lastBlockCommentEndIndex = text.lastIndexOf('*/', currentOffset);
+		return lastBlockCommentStartIndex > lastBlockCommentEndIndex;
```

Comment:
Ouch, I indeed missed that, thanks for pointing this out.

Those cases are IMHO rather difficult to handle with plain string magic - perhaps a better 
approach would be to use the go tooling for this. 
Perhaps we could leverage the [AST](https://golang.org/pkg/go/ast/#File) 
derived via [go Parser](https://golang.org/pkg/go/parser/#example_ParseFile) to find those single line / block comment as well as multiline string literals.

But I don't know how well the parser can deal with incomplete source code...
WDYT?